bin/meshroom_batch

```diff
@@ -157,6 +157,9 @@ with multiview.GraphModification(graph):
             if not result:
                 raise ValueError('Invalid param override: ' + str(p))
             node, t, param, value = result.groups()
+            multiValues = value.split(',')
```

Comment:
Should only be done if the attribute could contain multiple values, like groups or lists.

Reply:
See https://github.com/alicevision/Meshroom/blob/develop/meshroom/core/desc.py#L84